### PR TITLE
Changhiskhan/normalize uri

### PR DESCRIPTION
**Pull request by changhiskhan**

### CODE REVIEW

File: eto/internal/model/job.py [hunk 1]
```diff
@@ -12,8 +12,8 @@
 import sys  # noqa: F401
 
 from eto.internal.exceptions import ApiAttributeError
-from eto.internal.model_utils import (ApiTypeError,  # noqa: F401
-                                      ModelComposed, ModelNormal, ModelSimple,
+from eto.internal.model_utils import ApiTypeError  # noqa: F401
+from eto.internal.model_utils import (ModelComposed, ModelNormal, ModelSimple,
```
Comment (eddyxu) [inline eto/internal/model/job.py:16]: so `openapi generator` is not determinstic?

File: eto/resolver.py [hunk 2]
```diff
@@ -12,7 +13,7 @@ class EtoDatasetResolver(DefaultResolver):
 
     def resolve(self, dataset_name: str) -> Iterable[str]:
         dataset = eto.get_dataset(dataset_name)
-        return super().resolve(dataset.uri)
+        return super().resolve(_normalize_uri(dataset.uri))
```
Comment (eddyxu) [inline eto/resolver.py:16]: should we consider open this API as public api in rikai?

**Review by eddyxu — APPROVED** [review]
LGTM overall

File: eto/internal/model/job.py (hunk 1 above)
Comment (changhiskhan) [reply eto/internal/model/job.py:16]: i'm running black/isort manually

File: eto/resolver.py (hunk 2 above)
Comment (changhiskhan) [reply eto/resolver.py:16]: probably a good idea

**Pull request merged**